content/docs/components/heading/theming.mdx

```diff
@@ -2,3 +2,160 @@
 id: heading
 scope: theming
 ---
+
+The `Heading` component is a single part component. All of the styling is applied
+directly to the `h2` element.
+
+> To learn more about styling single part components, visit the [Component Style](/docs/styled-system/component-style#styling-single-part-components) page
+## Theming properties
+
+The properties that affect the theming of the `Heading` component are:
+
+- `variant`: The visual variant of the divider.
+- `colorScheme`: The color scheme of the divider.
+- `size`: The size of the divider. Defaults to `xl`.
+
+## Theming utilities
+
+- `defineStyle`: a function used to create style objects.
+- `defineStyleConfig`: a function used to define the style configuration for a single part component.
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+```
+
+## Customizing the default theme
+
+```js
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react'
+
+const custom = defineStyle({
+    color: "yellow.500",
+    border: "0px",
+    // let's also provide dark mode alternatives
+    _dark: {
+        color: 'yellow.300',
+    }
+})
+```
+After customizing the divider theme, we can import it in our theme file and add it in the components property:
+
+```js
+import { extendTheme } from '@chakra-ui/react'
+import { headingTheme } from './components/divider.ts'
+
+export const theme = extendTheme({
+    components: { Heading: headingTheme },
+})
+```
+
+> This is a crucial step to make sure that any changes that we make to the divider theme are applied.
+
+Every time you're adding anything new to the theme, you'd need to run the cli command to get proper autocomplete in your IDE. Here's how you can do that:
+
+```bash
+# run it once:
+npx chakra-cli tokens path/to/theme.ts
+# or run it in watch mode:
+npx chakra-cli tokens path/to/theme.ts --watch
+```
```

Comment:
This block has been changed. Check out the template to see the latest changes.

Reply:
This block is still not updated. It should just be this paragraph:

```markdown
Every time you're adding anything new to the theme, you'd need to run the CLI
command to get proper autocomplete in your IDE. You can learn more about the CLI
tool [here](/docs/styled-system/cli).
```

Reply:
Also, you should place this at the end of the `## Adding a custom variant` section.

Reply:
Still not changed.